### PR TITLE
Add crisprmatch

### PR DESCRIPTION
**Pull request by youqi1989**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: .circleci/common.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+MINICONDA_VER=4.5.4
```
Comment (bgruening) [inline .circleci/common.sh]: This file should not be here...

File: recipes/crisprmatchtest/build.sh [hunk 2]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+#$PYTHON setup.py install --single-version-externally-managed --record=record.txt
```
Comment (bgruening) [inline recipes/crisprmatchtest/build.sh]: why is this commented out?

File: recipes/crisprmatchtest/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+package:
+  name: crisprmatch
+  version: "v1.0"
```
Comment (bgruening) [inline recipes/crisprmatchtest/meta.yaml]: this should be 1.0

File: recipes/crisprmatchtest/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+source:
+  url: https://github.com/zhangtaolab/CRISPRMatch/archive/v1.0.tar.gz
+  sha256: 1aa546c7ca801467b29da2d50ec267ace7e96c5dc6a18aadfb05273d8e8a9bda
+
+build:
+  noarch: python
```
Comment (bgruening) [inline recipes/crisprmatchtest/meta.yaml]: you can include here `script: python -m pip install --no-deps --ignore-installed .`

File: recipes/crisprmatchtest/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+requirements:
+  build:
+    - git
```
Comment (bgruening) [inline recipes/crisprmatchtest/meta.yaml]: why it git here needed? Can you inlude pip and remove setuptools if it is not needed?

File: recipes/pyfaidx/0.4.7.1/build.sh [hunk 6]
```diff
@@ -1,9 +0,0 @@
-#!/bin/bash
```
Comment (bgruening) [inline recipes/pyfaidx/0.4.7.1/build.sh]: why have you removed this?

Comment (youqi1989) [reply recipes/pyfaidx/0.4.7.1/build.sh]: Hi, 'pyfaidx' is not the recipe I want to add. I have made some changes according to your commits, could you please review again?

File: .circleci/common.sh (hunk 1 above)
Comment (bgruening) [reply .circleci/common.sh]: ping

File: recipes/crisprmatchtest/meta.yaml (hunk 4 above)
Comment (bgruening) [reply recipes/crisprmatchtest/meta.yaml]: ping

File: recipes/crisprmatchtest/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+    - matplotlib
+    - pysam
+    - pandas
+    - argparse
+    - numpy
+    - pyqt 5.6.0
+
+test:
+  imports:
+    - pysam
```
Comment (bgruening) [inline recipes/crisprmatchtest/meta.yaml]: you don't test your python script/module here as it seems.

**Comment by daler** [comment]
Closing due to inactivity, please re-open if needed.

**Pull request closed without merging**